libraries/AP_Scripting/examples/Aerobatics/Trajectory/plane_aerobatics.lua

```diff
@@ -2305,7 +2306,7 @@ function load_trick(id)
       -- already have it
       return
    end
-   local fname = string.format("trick%u.txt", id)
+   local fname = string.format("scripts/trick%u.txt", id)
```

Comment:
better to try APM/scripts, scripts/ and current directory

Reply:
fixed in your PR....removing the lua changes